examples/rsp-next-ts/package.json

```diff
@@ -12,20 +12,20 @@
   },
   "type": "module",
   "dependencies": {
-    "@adobe/react-spectrum": "^3.22.0",
+    "@adobe/react-spectrum": "^3.26.0",
     "@react-spectrum/color": "^3.0.0-beta.16",
     "@react-spectrum/toast": "^3.0.0-alpha.0",
     "@spectrum-icons/illustrations": "^3.5.0",
     "@spectrum-icons/workflow": "^4.0.3",
-    "next": "13.2.4",
+    "next": "13.4.0",
```

Comment:
just a note, will need https://github.com/adobe/react-spectrum/pull/4486 to go in first since 13.4.0 for nextjs will run into that node version issue